packages/jetpack-blocks/webpack.config.js

```diff
@@ -0,0 +1,185 @@
+	const presetBetaBlocks = _.get( presetIndex, [ 'beta' ], [] );
+	const allPresetBlocks = [ ...presetBlocks, ...presetBetaBlocks ];
+
+	// Helps split up each block into its own folder view script
+	const viewBlocksScripts = allPresetBlocks.reduce( ( viewBlocks, block ) => {
+		const viewScriptPath = path.join( __dirname, 'src', 'blocks', block, 'view.js' );
+		if ( fs.existsSync( viewScriptPath ) ) {
+			viewBlocks[ block + '/view' ] = [ viewSetup, ...[ viewScriptPath ] ];
+		}
+		return viewBlocks;
+	}, {} );
+	//
+	// Combines all the different blocks into one editor.js script
+	const editorScript = [
+		editorSetup,
+		...blockScripts( 'editor', path.join( __dirname, 'src' ), presetBlocks ),
+	];
+	//
+	// Combines all the different blocks into one editor-beta.js script
+	const editorBetaScript = [
+		editorSetup,
+		...blockScripts( 'editor', path.join( __dirname, 'src' ), allPresetBlocks ),
+	];
+
+	const webpackConfig = {
+		bail: ! isDevelopment,
+		context: __dirname,
+		entry: {
+			editor: editorScript,
+			...( editorBetaScript && { 'editor-beta': editorBetaScript } ),
+			...viewBlocksScripts,
+		},
+		mode: isDevelopment ? 'development' : 'production',
+		devtool: process.env.SOURCEMAP || ( isDevelopment ? '#eval' : false ),
+		output: {
+			path: path.join( __dirname, 'dist' ),
+			filename: '[name].js',
+			libraryTarget: 'window',
+		},
+		optimization: {
+			minimize: shouldMinify,
+			minimizer: Minify( {
+				cache: process.env.CIRCLECI
+					? `${ process.env.HOME }/terser-cache`
+					: 'docker' !== process.env.CONTAINER,
+				parallel: workerCount,
+				sourceMap: Boolean( process.env.SOURCEMAP ),
+				terserOptions: {
+					ecma: 5,
+					safari10: true,
+					mangle: true,
+				},
+			} ),
+		},
+		module: {
+			rules: [
+				TranspileConfig.loader( {
+					workerCount,
+					configFile: path.join( __dirname, '..', '..', 'babel.config.js' ), // FIX ME
+					cacheDirectory: path.join( __dirname, 'build', '.babel-client-cache' ),
+					cacheIdentifier,
+					exclude: /node_modules\//,
+				} ),
+				SassConfig.loader( {
+					preserveCssCustomProperties,
+					includePaths: [ path.join( __dirname, 'client' ) ],
+					prelude: `@import '${ path.join(
+						__dirname,
+						'..',
+						'..',
+						'assets/stylesheets/shared/_utils.scss' // FIX ME
+					) }';`,
+				} ),
+				{
+					// TODO: turn in into a shard.
+					test: /\.(?:gif|jpg|jpeg|png|svg)$/i,
+					use: [
+						{
+							loader: 'file-loader',
+							options: {
+								name: '[name]-[hash].[ext]',
+								outputPath: 'images/',
+							},
+						},
+					],
+				},
+			],
+		},
+		resolve: {
+			extensions: [ '.json', '.js', '.jsx' ],
+			modules: [ 'node_modules' ],
+		},
+		node: false, // look into ?
+		plugins: _.compact( [
+			new webpack.DefinePlugin( {
+				'process.env.NODE_ENV': JSON.stringify( bundleEnv ),
+				global: 'window',
+			} ),
+			new webpack.IgnorePlugin( /^props$/ ), // look into ?
```

Comment:
This must be stale, isolating removal in https://github.com/Automattic/wp-calypso/pull/31525

Reply:
Confirmed and removed from main webpack config. Let's remove it.